lib/ui/components/Mocks/Sidebar/MocksSidebar.js

```diff
@@ -46,13 +47,29 @@ class MocksSidebar extends React.PureComponent {
   componentDidMount() {
     API.on(EVENTS.UPDATE_MOCK, this.reRender);
     API.on(EVENTS.UPDATE_GROUP, this.reRender);
+
+    document.addEventListener("keydown", this.onKeyDown);
   }
 
   componentWillUnmount() {
     API.off(EVENTS.UPDATE_MOCK, this.reRender);
     API.off(EVENTS.UPDATE_GROUP, this.reRender);
+
+    document.removeEndEventListener("keydown", this.onKeyDown);
   }
 
+  onKeyDown = (event) => {
+    if (event.keyCode === KEYCODES.BACKSPACE || event.keyCode === KEYCODES.DELETE) {
+        const selectedGroupsIds = this.props.selectedGroups.map(group => group.id);
```

Comment:
There is no need to do this logic yourself, we have State classes to handle these mutations so we can have them all in one place.

You can import `MocksState` which is the state for anything related directly to mocks (and groups) and use the `deleteSelected` method.